lib/workers/branch/index.js

```diff
@@ -44,6 +44,16 @@ async function processBranch(branchConfig, prHourlyLimitReached, packageFiles) {
     config.rebaseRequested = rebaseCheck(config, branchPr);
     logger.debug(`Branch pr rebase requested: ${config.rebaseRequested}`);
   }
+  if (
+    config.rebaseRequested &&
+    !config.rebaseConflictedPrs &&
```

Comment:
This is not relevant here

Reply:
Removed these changes and added changes to the appropriate file.